test/zdtm.py

```diff
@@ -28,7 +28,15 @@
 
 import yaml
 
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+os.chdir(SCRIPT_DIR)
+
+# Instead of passing the location of the image streamer binary as an argument
+# (e.g., --criu-image-streamer-bin), the caller can modify PATH to include the
+# binary location. We add a default location as well.
```

Comment:
What do you think about adding `make install` to criu-image-streamer?

Reply:
I merged your PR on criu-image-streamer.

I don't necessarily see a reason to change how:
1) Travis compiles criu-image-streamer (and doesn't install it)
2) CRIU locates the streamer binary in ../criu-image-streamer in addition to regular PATH
Maybe you do though. What do you think?

Reply:
No, it just makes it easier to run the tests locally when `criu-image-streamer` is installed.